test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -2564,6 +2502,26 @@ func initTestSchedulerForFrameworkTest(t *testing.T, testCtx *testutils.TestCont
 	return testCtx
 }
 
+// CleanupTestWithScheduler will close the scheduler and delete the nodes and pods.
```

Comment:
```suggestion
// cleanupTestSchedulerForFramework will close the scheduler and delete the nodes and pods.
```

Reply:
although it sounds a bit odd. Maybe `cleanupSchedulerTestCase`?

Reply:
Rename to `cleanupSchedulerFrameworkTest` vs `initTestSchedulerForFrameworkTest`.